tests/common.rs

```diff
@@ -62,7 +58,8 @@ impl ServerMoq {
                     packet_throughput = 0;
                 }
             }
-        })
+        });
```

Comment:
take it or leave it: remove the semi and you dont need to `let handle`

Reply:
There was a reason for that, but I can't remember it now, and it works fine without it.